src/zeit/cms/tagging/tag.py

```diff
@@ -136,12 +110,13 @@ def unique_id_to_tag(unique_id):
 
 class AbsoluteURL(zope.traversing.browser.absoluteurl.AbsoluteURL):
 
-    zope.component.adapts(Tag, zeit.cms.browser.interfaces.ICMSLayer)
+    zope.component.adapts(zeit.cms.tagging.interfaces.ITag,
+                          zeit.cms.browser.interfaces.ICMSLayer)
 
     def __str__(self):
         base = zope.traversing.browser.absoluteURL(
             zope.site.hooks.getSite(), self.request)
-        return base + '/++tag++' + self.context.code
+        return base + '/++tag++' + self.context.code.encode('unicode_escape')
```

Comment:
Damit kriegen wir Backslashes in die URL, das find ich nicht so wirklich hübsch. Spricht denn etwas gegen ganz handelsübliches urlencoding (`urllib.quote`)?